app/Http/Livewire/StreamList.php

```diff
@@ -20,14 +19,12 @@ public function render(): View
             ->when($this->isArchive, function($builder) {
                 $builder->finished()->fromLatestToOldest();
             }, function($builder) {
-                $builder->upcoming()->orderBy('scheduled_start_time');
+                $builder->upcoming()->fromOldestToLatest();
```

Comment:
I have added a scope to be in line with the other methods.

Reply:
Nice 👍